onnxruntime/core/providers/js/operators/where.cc

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "core/providers/js/js_kernel.h"
+
+namespace onnxruntime {
+namespace js {
+
+#define REG_ELEMENTWISE_KERNEL(OP_TYPE, VERSION, KERNEL_CLASS)      \
+  ONNX_OPERATOR_KERNEL_EX(                                          \
+      OP_TYPE,                                                      \
+      kOnnxDomain,                                                  \
+      VERSION,                                                      \
+      kJsExecutionProvider,                                         \
+      KernelDefBuilder()                                            \
+          .TypeConstraint("B",                                      \
+                          DataTypeImpl::GetTensorType<bool>())      \
```

Comment:
since B can only be tensor(bool), it's no need to set type constraint for B here. [spec](https://github.com/onnx/onnx/blob/main/docs/Operators.md#Where)

@pranavsharma correct me if I were wrong.